src/app/modules/admin-core/components/bread-crum/bread-crum.component.html

```diff
@@ -1,4 +1,4 @@
-<div *ngIf="edit != 'edit/:id'" class="pageName">
+<div *ngIf ="show" class="pageName">
```

Comment:
Why is this condition??

Reply:
In order to show the editing record in breadscrum